pkgs/data/themes/qtcurve/default.nix

```diff
@@ -1,32 +1,24 @@
-{ stdenv, fetchFromGitHub, cmake, extra-cmake-modules, pkgconfig, mkDerivation
+{ lib, fetchFromGitHub, cmake, extra-cmake-modules, pkgconfig, mkDerivation
 , gtk2, qtbase, qtsvg, qtx11extras # Toolkit dependencies
 , karchive, kconfig, kconfigwidgets, kio, frameworkintegration
 , kguiaddons, ki18n, kwindowsystem, kdelibs4support, kiconthemes
 , libpthreadstubs, pcre, libXdmcp, libX11, libXau # X11 dependencies
 , fetchpatch
 }:
 
-let
-  version = "1.9.1";
-in mkDerivation {
+mkDerivation {
   pname = "qtcurve";
-  inherit version;
+  version = "1.9.1";
   src = fetchFromGitHub {
     owner = "KDE";
     repo = "qtcurve";
-    rev = version;
-    sha256 = "0sm1fza68mwl9cvid4h2lsyxq5svia86l5v9wqk268lmx16mbzsw";
+    rev = "e534cd04e6acc90c105b492f3fcf99cc6155ba15";
```

Comment:
As an alternative to using unstable version you could add the following patch to release https://github.com/KDE/qtcurve/commit/44e2a35ebb164dcab0bad1a9158b1219a3ff6504

Reply:
Well it was the idea, I just have a few other patches that don't look very dangerous.
Do you think it is still important to add just one commit ?

Reply:
Do as you like, just don't forget about version :smiley:

Reply:
I too agree with @gnidorah on this - adding a patch is a bit more rigorous and a version change warrants a commit with `<old version> -> <new version>`.